book/advanced/proposer.md

```diff
@@ -42,6 +42,7 @@ Before starting the proposer, ensure you have deployed the L2 Output Oracle and
 
 | Parameter | Description |
 |-----------|-------------|
+| `PROVER_ADDRESS` | Address of the account that will be posting output roots to L1. This address is committed to when generating the aggregation proof to prevent front-running attacks. It can be different from the signing address if you want to separate these roles. Default: The address derived from the `PRIVATE_KEY` environment variable. |
```

Comment:
nit: Can you move this to the bottom of the environment variables (most users won't need to set this).